### PR TITLE
Update domains

### PR DESCRIPTION
**Pull request by AntoineMille**

### CODE REVIEW

File: dateofdelivery.php [hunk 1]
```diff
@@ -476,8 +476,8 @@ protected function _getDatesOfDelivery($id_carrier, $product_oos = false, $date
 		{
 			if ($elmt == 'l' || $elmt == 'F')
 			{
-				$date_minimal_string .= $this->l(date($elmt, $date_minimal_time));
-				$date_maximal_string .= $this->l(date($elmt, $date_maximal_time));
+				$date_minimal_string .= $this->trans(date($elmt, $date_minimal_time), array(), 'Modules.Dateofdelivery');
```
Comment (AlexEven) [inline dateofdelivery.php]: What's this? I'm not sure it should be translatable ^^

File: dateofdelivery.php [hunk 2]
```diff
@@ -476,8 +476,8 @@ protected function _getDatesOfDelivery($id_carrier, $product_oos = false, $date
...
+				$date_maximal_string .= $this->trans(date($elmt, $date_maximal_time), array(), 'Modules.Dateofdelivery');
```
Comment (AlexEven) [inline dateofdelivery.php]: Same here :)

File: dateofdelivery.php [hunk 3]
```diff
@@ -507,52 +507,52 @@ public function renderForm()
 		$fields_form = array(
 			'form' => array(
 				'legend' => array(
-					'title' => $this->l('Settings'),
+					'title' => $this->trans('Settings', array(), 'Modules.Dateofdelivery.Admin'),
```
Comment (AlexEven) [inline dateofdelivery.php]: Admin.Global

File: dateofdelivery.php [hunk 4]
```diff
@@ -507,52 +507,52 @@ public function renderForm()
...
+						'label' => $this->trans('Date format:'),
 						'name' => 'date_format',
-						'desc' => $this->l('You can see all parameters available at:').' <a href="http://www.php.net/manual/en/function.date.php">http://www.php.net/manual/en/function.date.php</a>',
-						),
+						'desc' => $this->trans('You can see all parameters available at:').' <a href="http://www.php.net/manual/en/function.date.php">http://www.php.net/manual/en/function.date.php</a>',
+						, array(), 'Modules.Dateofdelivery.Admin'),
 				),
 			'submit' => array(
-				'title' => $this->l('Save'),
+				'title' => $this->trans('Save', array(), 'Modules.Dateofdelivery.Admin'),
```
Comment (AlexEven) [inline dateofdelivery.php]: Admin.Actions

File: dateofdelivery.php [hunk 5]
```diff
@@ -588,13 +588,13 @@ public function renderAddForm()
 		$fields_form = array(
 			'form' => array(
 				'legend' => array(
-					'title' => $this->l('Settings'),
+					'title' => $this->trans('Settings', array(), 'Modules.Dateofdelivery.Admin'),
```
Comment (AlexEven) [inline dateofdelivery.php]: Admin.Global

File: dateofdelivery.php [hunk 6]
```diff
@@ -604,40 +604,40 @@ public function renderAddForm()
...
-						'label' => $this->l('Delivery between'),
+						'label' => $this->trans('Delivery between', array(), 'Modules.Dateofdelivery.Admin'),
 						'name' => 'minimal_time',
-						'suffix' => $this->l('day(s)'),
+						'suffix' => $this->trans('day(s)', array(), 'Modules.Dateofdelivery.Admin'),
 						),
 					array(
 						'type' => 'text',
-						'label' => $this->l(''),
+						'label' => $this->trans('', array(), 'Modules.Dateofdelivery.Admin'),
```
Comment (AlexEven) [inline dateofdelivery.php]: Shouldn't be translated?

File: dateofdelivery.php [hunk 7]
```diff
@@ -604,40 +604,40 @@ public function renderAddForm()
...
+									'name' => $this->trans('Delivery on Sunday', array(), 'Modules.Dateofdelivery.Admin'),
 									'val' => 1
 								),
 							),
 						)
 					)
 				),
 			'submit' => array(
-				'title' => $this->l('Save'),
+				'title' => $this->trans('Save', array(), 'Modules.Dateofdelivery.Admin'),
```
Comment (AlexEven) [inline dateofdelivery.php]: Admin.Actions

**Comment by maximebiloe** [comment]
Thanks @AntoineMille

**Pull request merged**